recipe/build.sh

```diff
@@ -25,7 +13,7 @@ INSTALL_CMD="make install"
 # tests are performed during building as they are not available in the
 # installed package.
 # Additional tests can be run with "make smallcheck" and "make bigcheck"
-TEST_CMD="eval cd tests && ${LIBRARY_SEARCH_VAR}=\"$PREFIX/lib\" make check-local && cd -"
+TEST_CMD="eval cd tests && make check-local && cd -"
```

Comment:
I think we can drop the `eval` too. This was originally used with the `LIBRARY_SEARCH_VAR`. However now that things have been shuffled and the `LIBRARY_SEARCH_VAR` dropped, I don't think `eval` serves any purpose here.